Update woocommerce-gateway-lightning.php more helpfull error message

renepickhardt opened the pull request:
made the error message if php-curl or php-gd are missing more explicit for inexperienced users by extenting the patch provided in https://github.com/ElementsProject/woocommerce-gateway-lightning/commit/ca40a13ed1289d95c0bd4d8305fdcd9ccc041b90
from issue: https://github.com/ElementsProject/woocommerce-gateway-lightning/issues/10

renepickhardt committed:
Update woocommerce-gateway-lightning.php

made the error message if php-curl or php-gd are missing more explicit for inexperienced users

shesek commented:
Thanks for the contribution, but I think the existing error message is clear enough and not sure this is worth the extra code. Should be easy enough to figure out which one is missing once you know that one of them are.

renepickhardt committed:
Update woocommerce-gateway-lightning.php

removed the case distinction about which plugin is missing. however I think the note for people running the plugin in managed hosting settings is still usefull

renepickhardt commented:
I removed the else statements. However I think particularly for inexperienced coders the hint that on a shared hosting plan (which many wordpress users are using) it might be a problem is helpful.

shesek commented:
I think a message like that is too specific. This isn't necessarily running on apache (could be running on a different http server, like nginx), and `php-gd`/`php-curl` support are quite common in shared hosting environments, so suggesting users to move away might be too extreme (a quick email to the technical support of their hosting provider might be able to solve that without switching servers).

renepickhardt committed:
Update woocommerce-gateway-lightning.php

made the error message more general

renepickhardt commented:
I changed the message once more. I am sure it'll be my last commit on this pull request (:

shesek commented:
I added this separately in 0f0ffa8225e3406ee17b791b454b10cc47d603f7, with a less verbose message and because this PR has 3 separate commits. Thanks Rene!

renepickhardt commented:
perfect thanks (: btw just setting up my lnd node. I probably fork your wp-woocommerce-gateway-lightning to a wp-woocommerce-gateway-lnd.

shesek commented:
Very cool! Would be great to have an lnd version of this. Good luck.
